src/main/java/com/faforever/api/security/FafUserDetailsService.java

```diff
@@ -23,6 +24,10 @@ public FafUserDetailsService(UserRepository userRepository) {
 
   @Override
   public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
-    return new FafUserDetails(userRepository.findOneByLoginIgnoreCase(username));
+    User uname = userRepository.findOneByLoginIgnoreCase(username); 
+    if (uname == null) {
+      throw new UsernameNotFoundException("User could not be found: "+username);
+    }
+    return new FafUserDetails(uname);
   }
-}
+} 
```

Comment:
In IntelliJ's commit dialog, please enable the following settings under "Before Commit":
* Reformat code
* Rearrange code
* Optimize imports

This will ensure that the code is properly formatted (which, currently, it's not :))